cms/templates/js/certificate-editor.underscore

```diff
@@ -0,0 +1,47 @@
+<form class="collection-edit-form certificate-edit-form">
+    <div aria-live="polite">
+    <% if (error && error.message) { %>
+        <div class="certificate-edit-error message message-status message-status error is-shown" name="certificate-edit-error">
+          <%= gettext(error.message) %>
+        </div>
+    <% } %>
+    </div>
+    <div class="wrapper-form">
+        <fieldset class="collection-fields certificate-fields">
+            <legend class="sr"><%= gettext("Certificate Information") %></legend>
+            <div class="input-wrap field text required add-collection-name add-certificate-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
+                <label for="certificate-name-<%= uniqueId %>"><%= gettext("Certificate Name") %></label>
+                <% if (!_.isUndefined(id)) { %>
+                    <span class="certificate-id">
+                        <span class="certificate-label"><%= gettext('Certificate ID') %></span>
+                        <span class="certificate-value"><%= id %></span>
+                    </span>
+                <% } %>
+                <input id="certificate-name-<%= uniqueId %>" class="collection-name-input input-text" name="certificate-name" type="text" placeholder="<%= gettext("Name of the certificate") %>" value="<%= name %>" aria-describedby="certificate-name-<%=uniqueId %>-tip">
```

Comment:
The name of the certificate should be removed and instead replaced with other general information fields. (If those fields are in a separate PR, it would be great to remove the need to set a name field entirely and hide this from the editing view.)